python/foglamp/services/core/server.py

```diff
@@ -214,8 +222,8 @@ def get_certificates():
         """
         # use pem file?
         # file name will be WHAT? *using foglamp for now*
-        cert = certs_dir + '/foglamp.cert'
-        key = certs_dir + '/foglamp.key'
+        cert = certs_dir + '/{}.cert'.format(cls.cert_file_name)
+        key = certs_dir + '/{}.key'.format(cls.cert_file_name)
         # remove these asserts and put in try-except block with logging
         assert os.path.isfile(cert)
         assert os.path.isfile(key)
```

Comment:
remove these assert and ` if not os.path.isfile(cert)` or key then fallback to foglamp.key and foglamp.cert.

now if foglamp.key and foglamp.cert does not exist raise RuntimeError